src/FactoryMuffin.php

```diff
@@ -362,4 +371,24 @@ private function loadDirectory($path)
             require $file->getPathName();
         }
     }
+
+    /**
+     * Translates a string with underscores
+     * into camel case (e.g. first_name -> firstName).
```

Comment:
Please don't allow the short description to overflow behond 1 line and 75 chars.